### PR TITLE
Update help.txt

### PR DESCRIPTION
**Pull request by punchitchewie**

### CODE REVIEW

File: help.txt [hunk 1]
```diff
@@ -1,2 +1,5 @@
 This is a help file
 Some text
+
+
+DFDFKHSKDJFHKJSDHFJKSHFDK
```
Comment (kairynners) [inline help.txt:5]: What is this supposed to be?

File: help.txt [hunk 2]
```diff
@@ -1,2 +1,5 @@
 This is a help file
 Some text
+
```
Comment (kairynners) [inline help.txt:3]: too many blank lines

Comment (punchitchewie) [reply help.txt:3]: helps for readability...

Comment (kairynners) [reply help.txt:3]: ok

**Pull request merged**